gst/nnstreamer/tensor_filter/tensor_filter.c

```diff
@@ -791,125 +588,9 @@ gst_tensor_filter_get_property (GObject * object, guint prop_id,
     case PROP_SILENT:
```

Comment:
Question: why this property remained in tensor-filter?

Reply:
silent requires access to `self` which is different type for `tensor_filter` and `tensor_filter_single`. (other properties do not require `self` but only `prop`)
The only way I could thing of moving this property to common function (`gst_tensor_common_filter_get_property`) would require to make the common function a macro.